refactor-group/refactor-platform-fe
# All environment variables for the dockerized application are set by the backend

## Description
This PR now ensures that all environment variables for the dockerized version of this application are set and passed from the backend/docker-compose.yaml.

#### GitHub Issue: None

### Changes
* All environment variables for the dockerized application are set and passed from the backend/docker-compose.yaml

### Screenshots / Videos Showing UI Changes (if applicable)


### Testing Strategy
1. Build the entire dockerized application from the backend
2. Ensure that the app functions as expected if you change environment variables in the backend's .env file


### Concerns
None
